src/utils/models/other.jl

```diff
@@ -373,6 +416,13 @@ struct MixedModel{O,FM<:AbstractModel} <: AbstractModel{O}
     end
 end
 
+"""
+    root(m::MixedModel)
+
+Return the `root` of model `m`.
+
+See also [`MixedModel`](@ref).
+"""
 root(m::MixedModel) = m.root
 
 isopen(::MixedModel) = isopen(root)
```

Comment:
isopen is still raising errors because of a name clash with Base.isopen, but will soon be renamed by @paraandrea.